### PR TITLE
feat: correctly add cognito

### PR DESCRIPTION
**Pull request by arthurlolico**

### CODE REVIEW

File: packages/auth/cloud/teste.spec.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+//import { template } from './cloudformation';
```
Comment (arantespp) [inline packages/auth/cloud/teste.spec.ts:1]: Remove this file

File: packages/auth/cloud/cloudformation.spec.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+test('test 1', () => {
```
Comment (arantespp) [inline packages/auth/cloud/cloudformation.spec.ts:1]: Fix tests.

**Review by arantespp — CHANGES_REQUESTED** [review]

**Comment by arantespp** [comment]
Closes #37

File: packages/auth/cloud/teste.spec.ts (hunk 1 above)
Comment (arthurlolico) [reply packages/auth/cloud/teste.spec.ts:1]: File removed.

File: packages/auth/cloud/cloudformation.spec.ts (hunk 2 above)
Comment (arthurlolico) [reply packages/auth/cloud/cloudformation.spec.ts:1]: How do I do that? 

expect(1).toBe(1)?

Comment (arantespp) [reply packages/auth/cloud/cloudformation.spec.ts:1]: No. You should test the method `getTemplate`.

```
expect(getTemplate())...
```

Comment (arthurlolico) [reply packages/auth/cloud/cloudformation.spec.ts:3]: Done.

File: packages/auth/cloud/cloudformation.spec.ts [hunk 3]
```diff
@@ -1,3 +1,5 @@
+import { getTemplate } from './cloudformation';
+
 test('test 1', () => {
```
Comment (arantespp) [inline packages/auth/cloud/cloudformation.spec.ts:3]: Fix the name of your test. You should always think, "How can I name my test to remember the context in five years?"

"test 1" isn't a good name. You could use something like, "getTemplate should return an object."

**Review by arantespp — CHANGES_REQUESTED** [review]

File: packages/auth/cloud/cloudformation.spec.ts (hunk 3 above)
Comment (arthurlolico) [reply packages/auth/cloud/cloudformation.spec.ts]: I get it. Fixed it now, Pedro.

**Comment by arantespp** [comment]
Remove the file `cloud/teste.spec.ts`

**Review by arantespp — APPROVED** [review]

**Pull request merged**